dotnet-desktop-guide/net/winforms/controls-design/snippets/usercontrol-overview/csharp/UserControl1.cs

```diff
@@ -0,0 +1,44 @@
+﻿//<default_event>
+using System.ComponentModel;
+
+namespace UserControlProject;
+
+[DefaultEvent(nameof(TextChanged))]
+public partial class UserControl1 : UserControl
+//</default_event>
+{
+    //<text_changed>
+    [Browsable(true)]
+    public new event EventHandler? TextChanged
```

Comment:
This snippet is being used in showing how to expose a "TextChanged" event to the designer. The old event doesn't show up in the designer on the derived type, so I did this to work around it and expose that underlaying event.

Reply:
The problem is that `TextChanged` is implemented on control, so it's a really untypical thing to do, even in a snippet, and I am concerned that this will be confusing. Could we come up with something other trivial, like a `TextLengthThreshold` property and a `TextLengthExceeded` event?